docs/advanced_reference/commands/autonomy_build-image.md

```diff
@@ -43,8 +56,54 @@ autonomy build-image [OPTIONS] [PUBLIC_ID_OR_HASH]
 
 ## Examples
 
+### To build a runtime image, run
+
+```bash
+$ cd /my_service
+$ autonomy build-image
+```
+
+or
+
+```bash
+$ autonomy build-image --service-dir /my_service
+```
+
+Builds the agent images for the service located in the directory `my_service`. 
+
+### To build a runtime image using a public ID
+
+```bash
+$ autonomy build-image <author>/<agent_package>:<package_hash>
+```
+
+**Note** The runtime images are build by fetching the packages from the IFPS registry, so before running the command make sure you have all of the required packages published on the IPFS registry.
+
+### To build a runtime image with a custom tag
+
+```bash
+$ autonomy build-image <author>/<agent_package>:<package_hash> --version <version>
+```
+
+This will tag the image using the `<version>` string rather than the `<package_hash>`, which would be `<author>/oar-<agent_package>:<version>` in this case
+
+### Use custom author name for tagging the image
+
+By default the command will use the author name from the local CLI config stored in `~/.aea` to tag the image. The user can override this using the `--image-author` flag.
```

Comment:
Write more clearly where this author comes from... "~/.aea" has not been addressed anywhere. It should refer to the author name specified with autonomy init.